apps/examples/Form/Basic/index.css

```
@@ -182,9 +181,9 @@
   justify-content: center;
   align-items: center;
 
-  width: 22px;
-  height: 22px;
-  border-radius: 6px;
+  width: 20px;
```

Stylelint (17.4.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)